spec/unit/middleware/requeue_exceptions_spec.rb

```diff
@@ -16,8 +16,8 @@ def around_perform(job)
         end
       end
 
-      let(:container) do 
-        container = container_class.new 
+      let(:container) do
+        container = container_class.new
```

Comment:
I'm actually not frustrated :).

As far as I know, everyone the team is on board with not committing trailing whitespace (particularly since we enforce it in some of the other repos).  Given that, I'm not convinced it would be worthwhile to spend retrospective time on it.  A global whitespace cleanup with lint check afterwards is fine, IMO.

> There is three setting to editor (not all editors support all of them): (a) ignore tailing whitespaces, (b) kill them, and (c) kill only in modified lines.

FWIW, There's a 4th that I use: a keybinding that allows me to strip whitespace when I want to.  I use it whenever I noticed trailing whitespace (which my editor highlights) but make sure to put it in its own commit.